.github/workflows/circleci.yml

```diff
@@ -1,7 +1,7 @@
 on: [status]
 jobs:
   circleci_artifacts_redirector_job:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
If this is not `ubuntu-22.04`, it is [`ubuntu-2204`](https://circleci.com/developer/machine/image/ubuntu-2004) :slightly_smiling_face:

Reply:
I'm not entirely sure if there's some magic happening on GHA before pointing it to CircleCI, but we could try to "break it both ways"; if it doesn't work one way, it surely is the other one! :laughing: